test/Microsoft.AspNetCore.Mvc.Formatters.Xml.Test/XmlDataContractSerializerOutputFormatterTest.cs

```diff
@@ -441,29 +438,6 @@ public void GetSupportedContentTypes_ReturnsSupportedTypes(Type type, object exp
             await Assert.ThrowsAsync(typeof(SerializationException),
                 async () => await formatter.WriteAsync(outputFormatterContext));
         }
-#else
-        // DataContractSerializer in CoreCLR does not throw if the declared type is different from the type being
```

Comment:
Ran this by @dougbu. They seem to have fixed the underlying issue so we no longer need the `#else`